homeassistant/components/alexa/handlers.py

```diff
@@ -1523,3 +1524,30 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return directive.response()
+
+
+@HANDLERS.register(("Alexa.CameraStreamController", "InitializeCameraStreams"))
+async def async_api_initialize_camera_stream(hass, config, directive, context):
+    """Process a InitializeCameraStreams request."""
+    entity = directive.entity
+    camera_component = hass.data.get(camera.DOMAIN)
+    camera_entity = camera_component.get_entity(entity.entity_id)
```

Comment:
Why do you need to get this? You can request stream with just entity ID ?

Reply:
Removed. Added `async_get_image_url()` helper to `camera`.